spec/integration/client_spec.rb

```diff
@@ -49,6 +49,9 @@ def push_test_images
       next if key.nil?
       expect(key).to eq r["tags"]
     end
+
+    tags = client.tags("registre")
```

Comment:
It is not, because I wanted to tag it differently than `registry`, which in this test already exists. So, my first thought was in catalan :wink: